src/Workspaces/Core/Desktop/Workspace/FileTextLoader.cs

```diff
@@ -101,6 +101,34 @@ protected virtual SourceText CreateText(Stream stream, Workspace workspace)
             return textAndVersion;
         }
 
+        internal override TextAndVersion LoadTextAndVersionSynchronously(Workspace workspace, DocumentId documentId, CancellationToken cancellationToken)
```

Comment:
retry is less a salient detail to caller than a bright beacon reminding them they are calling a synchronous method when they probably wanted to call the async version.